commands/peers/commands.go

```diff
@@ -0,0 +1,40 @@
+// Package peercommands implements the peer management commands
+package peercommands
+
+import (
+	"github.com/gluster/glusterd2/rest"
+)
+
+// Command is a holding struct used to implement the GlusterD Command interface
+type Command struct {
+}
+
+// Routes returns command routes. Required for the Command interface.
+func (c *Command) Routes() rest.Routes {
+	return rest.Routes{
+		rest.Route{
+			Name:        "GetPeer",
+			Method:      "GET",
+			Pattern:     "/peers/{peerid}",
```

Comment:
I will be important for some external consumers of our API that we provide access to peers/volumes using unique identifiers that remain unique throughout. We will be guaranteeing that names will be only unique at this point in time. A name can be removed and reused, or it could be changed.
For example, access via UUIDs will be beneficial for a system monitoring a volume. UUIDs will ensure that it is always monitoring the same volume, and not a new volume that could be recreated with the same name.